tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -1486,6 +1486,21 @@ public void MockDirectory_EnumerateDirectories_ShouldThrowWhenPathIsNotMocked()
             // Assert
             Assert.Throws<DirectoryNotFoundException>(action);
         }
+        
+        
+        [TestCase("Folder", @"Folder\SubFolder")]
+        [TestCase(@"Folder\", @"Folder\SubFolder")]
```

Comment:
I ran this test case on the real file system and it returned an empty collection.